provisioner_test.go

```diff
@@ -119,3 +123,75 @@ func TestNewDisembedderProvisioner(t *testing.T) {
 	assert.NoError(t, err)
 	testProvisionerSuccessful(t, *p, "linux", "amd64", DefaultVersionAstilectron, DefaultVersionElectron)
 }
+
+func TestZipShouldRemove(t *testing.T) {
+	l := log.New(log.Writer(), log.Prefix(), log.Flags())
+
+	appName := "Test"
+	dataDir, _ := filepath.Abs("./testdata/tmp")
+	vendorDir := filepath.Join(dataDir, "vendor") // https://github.com/CarsonSlovoka/go-astilectron/blob/e7796e5/paths.go#L56
+	astilectronDownloadDst := filepath.Join(vendorDir, fmt.Sprintf("astilectron-v%s.zip", DefaultVersionAstilectron))
+	astilectronDir := filepath.Join(vendorDir, fmt.Sprintf("astilectron"))
+	electronDownloadDst := filepath.Join(vendorDir, fmt.Sprintf("electron-%s-%s-v%s.zip", runtime.GOOS, runtime.GOARCH, DefaultVersionElectron))
+	electronDir := filepath.Join(vendorDir, fmt.Sprintf("electron-%s-%s", runtime.GOOS, runtime.GOARCH))
+
+	t.Log("Make sure the test directory doesn't exist.")
+	if err := os.RemoveAll(dataDir); err != nil && !os.IsNotExist(err) {
+		t.FailNow()
+	}
+
+	t.Logf("Create a directory for test only:\n%s\n", dataDir)
+	if err := os.MkdirAll(dataDir, os.FileMode(0666)); err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	defer func() {
+		t.Log("Remove the test directory after the test was done.")
+		if err := os.RemoveAll(dataDir); err != nil {
+			t.Fatalf(err.Error())
+		}
+	}()
+
+	{
+		t.Log("common process")
+		a, err := New(l, Options{
```

Comment:
you can provide `astikit.AdaptTestLogger(t)` to `New()` instead of `l`

Reply:
- [New(&testingHelper.StdLogger{T: t}, o)](https://github.com/asticode/go-astilectron/pull/358/commits/ae08b36c5a8f4b2b68add1fd5808ad5e8201c78a#diff-f5c6513a93e0d3a0d10fb30654a66c3ab4b7f05ed5b4dd9dd10ef379a14496adR134)
- [AdaptTestLogger](https://github.com/asticode/go-astilectron/pull/358/commits/ae08b36c5a8f4b2b68add1fd5808ad5e8201c78a#diff-e07ecb7160cdf594e45e1f42ae28e80f629d1d3d9689998b10521ed9f8e59bc3R1-R23)

----

I'm not sure how to do that. It looks like it should be written in [asticode/go-astikit/logger.go](https://github.com/asticode/go-astikit/blob/master/logger.go).

Reply:
you need to replace `go-astikit` version in `go.mod` from `0.15.0` to `0.29.1`, then run `go mod tidy`. Then you'll be able to use [this function](https://github.com/asticode/go-astikit/blob/master/logger.go#L183) with `astikit.AdaptTestLogger(t)`

Reply:
Wow! You are right 👍👍